### DISCUSSION TITLE
Is there an option can rotate the whole chart for 90 degree?

### DISCUSSION THREAD

**Discussion by stalkercn** [patrykandpatrick/vico#983] [Questions] [answered]
### Question

From:
![1](https://github.com/user-attachments/assets/46055957-fb44-4955-9258-3d219d62786e)
To:
![2](https://github.com/user-attachments/assets/e484561f-445c-47dd-8f94-f2abf0f7bef7)

### Vico version(s)

2.0.0-beta3

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment, marked as answer]
Hello, @stalkercn. Yes, just like with any other composable, you can do this via modifiers.
